splinepy/helpme/extract.py

```diff
@@ -160,10 +160,15 @@ def faces(
             boundaries = spline.boundary_multipatch()
         else:
             boundaries = Multipatch(splines=spline.extract.boundaries())
-
-        n_faces = (resolution - 1) ** 2
-        vertices = resolution**2
-        f_loc = connec.make_quad_faces(enforce_len(resolution, 2))
+        # The following block assumes a integer resolution but might get a list
+        # as input. In that case, we take the max of the list and use it as
+        # resolution.
+        # I am not sure if this is correct or if we should instead rework the
+        # following lines so that list resolution is supported.
+        res = resolution if isinstance(resolution, int) else max(resolution)
```

Comment:
This line fixed the issue. @j042 Would you look at this and tell me if this is a correct fix or if we should do something different to fix the issue?

This was (for me at least) the simplest fix, but it is not really that clean.

Reply:
I'd rather have a deeper look. ever since #194, resolutions got mixed up.